packages/salesforcedx-vscode-lwc/package.json

```diff
@@ -99,8 +99,8 @@
       "main": "dist/index.js",
       "dependencies": {
         "applicationinsights": "1.0.7",
-        "@salesforce/core": "^3.34.8",
-        "@salesforce/source-tracking": "3.1.1",
+        "@salesforce/core": "^4.0.1",
+        "@salesforce/source-tracking": "4.1.0",
```

Comment:
is this ok as 4.1.0 or was it meant to be 4.1.3 as everywhere else?

Reply:
Thanks for pointing it out. It should be 4.1.3 like everywhere else.